### ISSUE TITLE
flannel in error and etcd can't get its tls certificate

### ISSUE THREAD

**Issue by qroussel** [juju-solutions/bundle-canonical-kubernetes#288]
Hello,

I've try to deploy canocical Kubernetes, and several time it gets stuck at a point where etcd can't get its certificate from easyrsa and flannel is in error with the message : "hook failed: "etcd-relation-changed" for flannel:etcd"

I've try to redeploy it several time today and I got stuck at the exact same point.

Here is my Juju status : 

![image](https://cloud.githubusercontent.com/assets/28761822/26165264/a045b3b8-3b2f-11e7-9426-f55332a33cd0.png)

Thank you in advance !

**Comment by chuckbutler** [comment]
Greetings, this was patched as of etcd-34.  I just pushed the updated bundles to the stable channel in the charm store about 45 minutes ago. Really sorry you ran into this, a breaking change was introduced from snapd that expanded the fields we were using to parse out the version, and it's not in a machine readable format. We've refactored the version parser to be much less brittle which would help mitigate these problems moving forward.

Would yo mind terribly redeploying again and reporting back if you have any issues?
